pkgs/os-specific/linux/criu/default.nix

```diff
@@ -1,26 +1,27 @@
 { stdenv, fetchurl, protobuf, protobufc, asciidoc
-, xmlto, utillinux, docbook_xsl, libpaper, libnl, libcap, pkgconfig
-, python }:
+, xmlto, utillinux, docbook_xsl, libpaper, libnl, libcap, libnet, pkgconfig
+, python, git }:
 
 stdenv.mkDerivation rec {
   name    = "criu-${version}";
-  version = "2.0";
+  version = "2.12.1";
 
   src = fetchurl {
     url    = "http://download.openvz.org/criu/${name}.tar.bz2";
-    sha256 = "1zqqshslcf503lqip89azp1zz0i8kb7v19b3dyp52izpak62c1z8";
+    sha256 = "18m0sjgcfvzc86w49fd3kxw145nmrsvc5w7zf42nxdiklmszbr1k";
   };
 
   enableParallelBuilding = true;
   nativeBuildInputs = [ pkgconfig docbook_xsl ];
-  buildInputs = [ protobuf protobufc asciidoc xmlto libpaper libnl libcap python ];
+  buildInputs = [ protobuf protobufc asciidoc xmlto libpaper libnl libcap libnet python git ];
```

Comment:
As far as I can tell the git here is not actually used, presumably because of the substitution in the patch phase… At least I was able to build it OK without git and
```
nix-store -q --references /nix/store/609y5nypwvwmm48ivj3lq7vrpp3px7fm-criu-2.12.1
```
doesn't show any reference to the git package.

Reply:
Ah, I see now, I do actually substituteInPlace the git call from Makefile so it's not needed. Will patch that.